core/src/test/scala/kafka/server/LocalLeaderEndPointTest.scala

```diff
@@ -271,9 +271,10 @@ class LocalLeaderEndPointTest extends Logging {
                             origin: AppendOrigin = AppendOrigin.CLIENT,
                             requiredAcks: Short = -1): CallbackResult[PartitionResponse] = {
     val result = new CallbackResult[PartitionResponse]()
-    def appendCallback(responses: scala.collection.Map[TopicIdPartition, PartitionResponse]): Unit = {
-      val response = responses.get(partition)
-      assertTrue(response.isDefined)
+    def appendCallback(responses: java.util.Map[TopicIdPartition, PartitionResponse]): Unit = {
```

Comment:
Given that we already have an import alias for `java.util.Map`, we could reuse it.
```suggestion
    def appendCallback(responses: JMap[TopicIdPartition, PartitionResponse]): Unit = {
```